.github/workflows/test.yml

```diff
@@ -38,11 +38,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
         airflow-version: ["2.4", "2.5", "2.6", "2.7", "2.8", "2.9"]
         exclude:
           - python-version: "3.11"
             airflow-version: "2.4"
+          - python-version: "3.12"
```

Comment:
@pankajastro could you add a comment/link explaining why we're excluding these versions of Airflow?

Reply:
Added PTAL 29b9f807501724412e9a30984464b4e9b96dbe4f

Reply:
Thanks, @pankajastro !